README.md

```diff
@@ -0,0 +1,183 @@
+3: -- run_test.py: invoking following command in '/home/jwang-robot/ros2_ws/src/ros2/message_filters':
+3:  - /home/jwang-robot/ros2_ws/build/message_filters/message_filters-test_chain --gtest_output=xml:/home/jwang-robot/ros2_ws/build/message_filters/test_results/message_filters/message_filters-test_chain.gtest.xml
+3: [==========] Running 8 tests from 1 test case.
+3: [----------] Global test environment set-up.
+3: [----------] 8 tests from Chain
+3: [ RUN      ] Chain.simple
+3: [       OK ] Chain.simple (0 ms)
+3: [ RUN      ] Chain.multipleFilters
+3: [       OK ] Chain.multipleFilters (0 ms)
+3: [ RUN      ] Chain.addingFilters
+3: [       OK ] Chain.addingFilters (1 ms)
+3: [ RUN      ] Chain.inputFilter
+3: [       OK ] Chain.inputFilter (0 ms)
+3: [ RUN      ] Chain.nonSharedPtrFilter
+3: [       OK ] Chain.nonSharedPtrFilter (0 ms)
+3: [ RUN      ] Chain.retrieveFilter
+3: [       OK ] Chain.retrieveFilter (0 ms)
+3: [ RUN      ] Chain.retrieveFilterThroughBaseClass
+3: [       OK ] Chain.retrieveFilterThroughBaseClass (0 ms)
+3: [ RUN      ] Chain.retrieveBaseClass
+3: [       OK ] Chain.retrieveBaseClass (0 ms)
+3: [----------] 8 tests from Chain (1 ms total)
+3: 
+3: [----------] Global test environment tear-down
+3: [==========] 8 tests from 1 test case ran. (1 ms total)
+3: [  PASSED  ] 8 tests.
+3: -- run_test.py: return code 0
+3: -- run_test.py: inject classname prefix into gtest result file '/home/jwang-robot/ros2_ws/build/message_filters/test_results/message_filters/message_filters-test_chain.gtest.xml'
+3: -- run_test.py: verify result file '/home/jwang-robot/ros2_ws/build/message_filters/test_results/message_filters/message_filters-test_chain.gtest.xml'
+ 3/10 Test  #3: message_filters-test_chain .....................   Passed    0.09 sec
+test 4
+...
+...
+...
+100% tests passed, 0 tests failed out of 10
+
+Label Time Summary:
+gtest     =  16.15 sec (9 tests)
+pytest    =   0.54 sec (1 test)
+
+Total Test time (real) =  16.69 sec
+
+```
+
+- Security check
+```
+$./build/message_filters/message_filters-test_fuzz
+
+[==========] Running 3 tests from 3 test cases.
+[----------] Global test environment set-up.
+[----------] 1 test from TimeSequencer
+[ RUN      ] TimeSequencer.fuzz_sequencer
+[       OK ] TimeSequencer.fuzz_sequencer (5118 ms)
+[----------] 1 test from TimeSequencer (5118 ms total)
+
+[----------] 1 test from TimeSynchronizer
+[ RUN      ] TimeSynchronizer.fuzz_synchronizer
+[       OK ] TimeSynchronizer.fuzz_synchronizer (5013 ms)
+[----------] 1 test from TimeSynchronizer (5013 ms total)
+
+[----------] 1 test from Subscriber
+[ RUN      ] Subscriber.fuzz_subscriber
+[       OK ] Subscriber.fuzz_subscriber (5010 ms)
+[----------] 1 test from Subscriber (5010 ms total)
+
+[----------] Global test environment tear-down
+[==========] 3 tests from 3 test cases ran. (15142 ms total)
+[  PASSED  ] 3 tests.
+```
+Any security issue should be reported using process at https://01.org/security."
```

Comment:
Is it possible to add one bullet - `Known Issues` to allow the other to get aware of the legacy work to do or some issues that we'll address in the near future if we actually have. thanks !

Reply:
@jwang11 , pls. remove the ' " ' at the end of last line in README.